components/global/portal/io.cellery.observability.ui/src/utils/api/httpUtils.test.js

```diff
@@ -186,7 +186,7 @@ describe("HttpUtils", () => {
 
         it("should add application/json header", async () => {
             axios.mockImplementation((config) => {
-                expect(Object.keys(config)).toHaveLength(3);
+                expect(Object.keys(config)).toHaveLength(4);
```

Comment:
Let's another check for the withCredentials value in the config. Please fix in all the unit tests.